tests/system/python/packages/data/package_list.json

```diff
@@ -0,0 +1,16 @@
+{
+	"p0": [{
+		"south": ["foglamp-south-modbus", "foglamp-south-sinusoid", "foglamp-south-opcua"],
+		"north": ["foglamp-north-http-north", "foglamp-north-kafka"],
+		"filter": ["foglamp-filter-python35"],
+		"notificationDelivery": ["foglamp-notify-python35"],
+		"notificationRule": ["foglamp-rule-outofbound"]
+	}],
+	"p1": [{
+		"south": ["foglamp-south-randomwalk", "foglamp-south-playback"],
+		"north": ["foglamp-north-httpc"],
+		"filter": ["foglamp-filter-threshold"],
+		"notificationDelivery": ["foglamp-notify-slack"],
+		"notificationRule": ["foglamp-rule-average"]
+	}]
```

Comment:
@MarkRiddoch @praveen-garg Please review this list. By default `P0` will run

Reply:
-- south

- P0
envirophat
game
sinusoid
http
modbus-c
opcua [X buster]
benchmark
playback
mqtt-sparkplug
randomwalk
wind-turbine
FlirAX8
raspi-classify   [x RPi4]
dnp3
phidget

- P1
pt100
am2315
cc2650
dht11
sensehat
dht
ina219
usb4704
sensorphone
openweathermap
coap
random
expression
image-classifier
beckhoff

- P2
systeminfo
modbustcp
roxtec
csv
csv-async

- CORAL only platform - for coral P0 :)
people
coral-classify
coral-people-image
coral-object-detection


-- north
- P0
httpc
kafka
opcua (X buster)

- P1
http
thingspeak

-- filter
- P0
python35
expression
delta
metadata
rms
fft
change

- P1
Flir-Validity (may be P0)  
scale
asset
scale-set
rate
rms-trigger
threshold

- P2
python27


-- notify rule 
- P0
simple-expression
average

- P1
outofbound

-- notify delivery
- P0
asset
email
python35

- P1
slack

- P2
telegram
ifttt
alexa-notifyme
blynk
google-hangouts

Reply:
That looks like a pretty good list. It might be worth using fogamp-notify-asset as that is also an easy one to verify in an end to end test.